erigon-lib/diagnostics/entities.go

```diff
@@ -155,6 +160,40 @@ type CPUInfo struct {
 	Mhz       float64 `json:"mhz"`
 }
 
+type BlockMetrics struct {
+	Header         *list.List
+	Bodies         *list.List
+	ExecutionStart *list.List
+	ExecutionEnd   *list.List
+	Production     *list.List
+}
+
+func NewBlockMetrics() BlockMetrics {
+	return BlockMetrics{
```

Comment:
general question: are we planning on adding these metrics to prometheus? has this been discussed between you & mark? curious what you guys have decided but my opinion is that these are quite valuable metrics to see from the "historical" point of view - ie how these metrics have changed for a node over time